engine/utils/cortexso_parser.h

```diff
@@ -6,6 +6,7 @@
 #include <services/download_service.h>
 #include <nlohmann/json.hpp>
 #include "httplib.h"
+#include "utils/file_manager_utils.h"
 
 namespace cortexso_parser {
```

Comment:
What are we trying to refer to by naming this `cortexso_parser`? 
We might need a better name for this - i.e. this refers to a Cortex Model Repo (i.e. Git repo, tag-based)

Reply:
I agree. But I will KIV this. Ideally, we will have multiple url parsers.